english/fork-jam/altar-smash/altar-smash.md

```diff
@@ -0,0 +1,5 @@
+You smash the graham cracker altar, and the chalice and biscuit debris vanish in a puff of powdered sugar.
```

Comment:
It looks like there is no link to this new file.  Did you forget to commit that change?  Make sure you create (and commit) an option for each new "page" you create.